requirements.txt

```diff
@@ -16,6 +16,6 @@ typer>=0.9.0
 rich>=13.5.3
 torch==1.13.0 #+cu116
 --find-links https://download.pytorch.org/whl/torch_stable.html
-transformers>=4.23.1
-sacrebleu>=2.3.1
-sentencepiece>=0.1.97
+transformers==4.23.1
+sacrebleu==2.3.1
+sentencepiece==0.1.97
```

Comment:
What are the consequences of fixing those requirements?
I would prefer that, if it is not necessary to fix them, that we leave them as they were before.

However, it's not critical, as you will eventually outsource the requirements to each plugin. If you don't want to do it, no problem.

Reply:
This changes where made by @cristian-tamblay.